### PR TITLE
INFRA: Initialize API Project

### PR DESCRIPTION
**Pull request by Spectralgo**

### CODE REVIEW

File: Refugee.Core.Api/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+            
+            services.AddSwaggerGen(options =>
+            {
+                options.SwaggerDoc(
+                    name: "v1",
```
Comment (hassanhabib) [inline Refugee.Core.Api/Startup.cs:29]: Put a new line between `name: "v1"` and the line below.

File: global.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (hassanhabib) [inline global.json:1]: I don't think we need this file at all.

File: Refugee.Core.Api/Controllers/HomeController.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+// -------------------------------------------------------
+
+using Microsoft.AspNetCore.Mvc;
+using RESTFulSense.Controllers;
+
+namespace Refugee.Core.Api.Controllers
+{
+    
+    [Route("api/[controller]")]
+    [ApiController]
```
[the remark this answers is not in the document]
Comment (Spectralgo) [reply Refugee.Core.Api/Controllers/HomeController.cs:12]: Done

File: Refugee.Core.Api/Startup.cs (hunk 1 above)
Comment (Spectralgo) [reply Refugee.Core.Api/Startup.cs:29]: Done

File: global.json (hunk 2 above)
Comment (Spectralgo) [reply global.json]: Done

File: Refugee.Core.Api/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            if (webHostEnvironment.IsDevelopment())
+            {
+                applicationBuilder.UseDeveloperExceptionPage();
+                applicationBuilder.UseSwagger();
+                
+                applicationBuilder.UseSwaggerUI(options => 
+                    options.SwaggerEndpoint(
+                        url: "/swagger/v1/swagger.json",
+                        name: "Refugee.Core.Api v1"));
+
```
Comment (ElbekDeveloper) [inline Refugee.Core.Api/Startup.cs:52]: Remove this line. We don't need it...

File: Refugee.Core.Api/Startup.cs (hunk 1 above)
Comment (ElbekDeveloper) [reply Refugee.Core.Api/Startup.cs:29]: https://github.com/hassanhabib/Taarafo.Core/blob/master/Taarafo.Core/Startup.cs can you take example from this

**Review by ElbekDeveloper — CHANGES_REQUESTED** [review]

**Review by ElbekDeveloper — APPROVED** [review]

**Pull request merged**